.github/workflows/ci.yml

```diff
@@ -26,30 +25,52 @@ jobs:
+        run: |
+          npx ajv-cli test -s schemas/dbt_project-1.5.json -d tests/1.5/valid/dbt_project.yml --valid
+          npx ajv-cli test -s schemas/dbt_project-1.5.json -d tests/1.5/invalid/dbt_project.yml --invalid
+      - name: Validate resource yml sample files for dbt version 1.5
+        run: |
+          npx ajv-cli test -s schemas/dbt_yml_files-1.5.json -d tests/1.5/valid/schema.yml --valid
+          npx ajv-cli test -s schemas/dbt_yml_files-1.5.json -d tests/1.5/invalid/schema.yml --invalid
+      - name: Validate packages.yml sample files for dbt version 1.5
+        run: |
+          npx ajv-cli test -s schemas/packages-1.5.json -d tests/1.5/valid/packages.yml --valid
+          npx ajv-cli test -s schemas/packages-1.5.json -d tests/1.5/invalid/packages.yml --invalid
+      - name: Validate selectors.yml sample files for dbt version 1.5
         run: |
-          npx ajv-cli test -s schemas/dbt_project.json -d tests/valid/dbt_project.yml --valid
-          npx ajv-cli test -s schemas/dbt_project.json -d tests/invalid/dbt_project.yml --invalid
-      - name: Validate resource yml sample files
+          npx ajv-cli test -s schemas/selectors-1.5.json -d tests/1.5/valid/selectors.yml --valid
+          npx ajv-cli test -s schemas/selectors-1.5.json -d tests/1.5/invalid/selectors.yml --invalid
+
+      - name: Validate dbt_project.yml sample files for dbt version 1.6
+        run: |
+          npx ajv-cli test -s schemas/dbt_project-1.6.json -d tests/1.6/valid/dbt_project.yml --valid
+          npx ajv-cli test -s schemas/dbt_project-1.6.json -d tests/1.6/invalid/dbt_project.yml --invalid
+      - name: Validate resource yml sample files for dbt version 1.6
         run: |
-          npx ajv-cli test -s schemas/dbt_yml_files.json -d tests/valid/schema.yml --valid
-          npx ajv-cli test -s schemas/dbt_yml_files.json -d tests/invalid/schema.yml --invalid
-      - name: Validate packages.yml sample files
+          npx ajv-cli test -s schemas/dbt_yml_files-1.6.json -d tests/1.6/valid/schema.yml --valid
+          npx ajv-cli test -s schemas/dbt_yml_files-1.6.json -d tests/1.6/invalid/schema.yml --invalid
+      - name: Validate packages.yml sample files for dbt version 1.6
         run: |
-          npx ajv-cli test -s schemas/packages.json -d tests/valid/packages.yml --valid
-          npx ajv-cli test -s schemas/packages.json -d tests/invalid/packages.yml --invalid
-      - name: Validate selectors.yml sample files
+          npx ajv-cli test -s schemas/packages-1.6.json -d tests/1.6/valid/packages.yml --valid
+          npx ajv-cli test -s schemas/packages-1.6.json -d tests/1.6/invalid/packages.yml --invalid
+      - name: Validate selectors.yml sample files for dbt version 1.6
+        run: |
+          npx ajv-cli test -s schemas/selectors-1.6.json -d tests/1.6/valid/selectors.yml --valid
+          npx ajv-cli test -s schemas/selectors-1.6.json -d tests/1.6/invalid/selectors.yml --invalid
```

Comment:
Two thoughts: 
- A single-dimension matrix might dry this up a bit: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs
- Is it worth testing that
  - a valid 1.6 file is rejected by the 1.5 validator, and
  -  a valid 1.5 file is still accepted by the 1.6 validator? (if we only do one of these two, it should be this one; they should all be backwards compatible, but they're not guaranteed to be forwards-incompatible)